README.md

```diff
@@ -27,6 +27,16 @@ When you deploy this website to Azure you can define the [Backend](https://githu
 
 > Note: you can change the InstrumentationKey of the **Application Insight** that is configured by default.
 
+In order to change the backend, set `apiBaseUrl` parameter on the ARM template provided to the url where your aks is configured.
+
+e.g. In order to know your aks route, if configured with `addon-http-application-routing` ingress class (as it's by default) you could run 
```

Comment:
AKS in capitals. 
Review the writing in the if configured sentence. I don't understand it.